NORDIC Embedded Software/nrf52 Patient Monitoring Device/ACC_file.c

```diff
@@ -16,18 +16,15 @@ STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY
 EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 */
 
-#include <stdlib.h>
-#include <stdio.h>
+
 #include "ACC_file.h"
 #include "config.h"
 
 void LSM303D_set_mode(uint8_t REGISTER)
```

Comment:
It is better to use lowercase in names. Capital letters are reserved for defines